mypaint-brush.c

```diff
@@ -22,6 +22,18 @@
 #include <math.h>
 #include <assert.h>
 
+#define GETTEXT_PACKAGE "libmypaint"
+
+#ifdef HAVE_GETTEXT
```

Comment:
In my opinion, it should be a separate file header to include, since we can avoid duplication code in other .c files that require translations messages too. When this would be merged, I can update other strings over all the code remaining ;)